procedures/foreman/branch.md.erb

```diff
@@ -12,98 +14,114 @@
     - [ ] `bundle exec rake pin_modules && sed -i '/Puppetfile.lock/d' .gitignore && bundle exec librarian-puppet install && git add Puppetfile*`
   - [ ] [foreman-selinux](https://github.com/theforeman/foreman-selinux)
   - [ ] [smart-proxy](https://github.com/theforeman/smart-proxy)
-- Branch foreman-packaging:
-  - [ ] Create rpm/<%= release %> and update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}` and `repoclosure/*.conf`
-  - [ ] Create deb/<%= release %>
-  - [ ] Update `.github/PULL_REQUEST_TEMPLATE.md` (by adding `[ ] <%= release %>` to it) in `master`
-- Bump versions to <%= develop %>-develop:
+
+The next step should only be done after all branching, including packaging, has completed.
+
+- [ ] Bump versions to <%= develop %>-develop
   `echo <%= develop %>.0-develop > VERSION`
   - [ ] [foreman](https://github.com/theforeman/foreman)
   Also change package.json version field to <%= develop %>.0
   - [ ] [foreman-installer](https://github.com/theforeman/foreman-installer)
   - [ ] [foreman-selinux](https://github.com/theforeman/foreman-selinux)
   - [ ] [smart-proxy](https://github.com/theforeman/smart-proxy)
-- Update foreman-packaging to <%= develop %>:
+
+# Preparing build systems
+
+## Release engineer
+
+- [ ] Update foreman-packaging rpm/develop to ensure nightlies build:
   - [ ] rpm/develop:
     - [ ] Update the build tag: `sed -i 's/fm<%= release.tr('.', '_') %>/fm<%= develop.tr('.', '_') %>/g' rel-eng/{releasers.conf,tito.props}`
     - [ ] Set to version `<%= develop %>.0`, reset release to `1` in `packages/foreman/foreman{,-{installer,proxy,release,selinux}}/*.spec` and create a changelog using `obal changelog --message '- Bump version to <%= develop %>-develop' foreman{,-{installer,proxy,release,selinux}}`
   - [ ] deb/develop: `scripts/changelog.rb -v <%= develop %>.0-1 -m "Bump changelog to <%= develop %>.0 to match VERSION" debian/*/*/changelog`
+- [ ] Prepare build systems for <%= release %> release:
+  - [ ] foreman-packaging's rpm/<%= release %>
+    - [ ] Update `packages/foreman/foreman-release/foreman.gpg`, `mock/*.cfg`, `package_manifest.yaml`, `rel-eng/{releasers.conf,tito.props}` and `repoclosure/*.conf`
+  - [ ] Create mash configuration on Koji:
+    ```
+    bundle exec ./tools.rb mash-scripts configs/foreman/<%= release %>.yaml
+    sed -i '/strict_keys/ s/True/False/' mash_scripts/foreman/<%= release %>.0/foreman-plugins-<%= release %>-*.mash
+    scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
+    ```
+  - [ ] Update [mash scripts](https://github.com/theforeman/foreman-infra/tree/master/koji/) and deploy them to Koji:
+    - [ ] Verify main mash scripts (`foreman-mash-split.py`, `foreman-rails-mash-split.py` and `foreman-plugin-mash-split.py`) don't need any version-specific updates
+    - [ ] Update dists in client mash script (`foreman-client-mash-split.py`)
```

Comment:
Not a blocker, but was changed and is now only https://github.com/theforeman/foreman-infra/blob/master/koji/collection-mash-split.py

Reply:
i'll leave this to a separate pr